server/routes/money.js

```diff
@@ -1,7 +1,46 @@
 const { path } = require('ramda');
 const express = require('express');
+const { subDays } = require('date-fns');
+const { formatBalanceOrDefault, formatPrisonName } = require('../utils/string');
+const { formatDateOrDefault } = require('../utils/date');
 
-const createMoneyRouter = ({ hubContentService, offenderService }) => {
+function formatTransaction(t) {
```

Comment:
could possibly extract a view kind of object that creates your `data` structure and remove a lot of presentation logic out of here